frontend/src/static/js/v_authorship.js

```diff
@@ -170,21 +171,34 @@ window.vAuthorship = {
 
       files.forEach((file) => {
         const lineCnt = file.authorContributionMap[this.info.author];
-        if (lineCnt) {
-          totalLineCount += lineCnt;
-          const out = {};
-          out.path = file.path;
-          out.lineCount = lineCnt;
-          out.fileType = file.fileType;
 
+        // skip if binary file not touched by author
+        if (file.isBinary
+          && !file.binaryFileAuthors.map((author) => author.gitId).includes(this.info.author)) {
```

Comment:
Do we really need this `binaryFIleAuthors`? Can we just use the `authorContributionMap` to find all the authors made change to this binary file?

Reply:
Thanks for the suggestion! I never thought of that. Will change to use `authorContributionMap` instead.